README.md

```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
- *  Tools: Provides tools used for development, such as an integration into the Thread Test Harness and a Sniffer
- *  Small memory footprint
+<a href="https://codelabs.developers.google.com/codelabs/openthread-simulation/index.html">
+<img src="doc/images/ot-codelab.png" alt="OpenThread Codelab" width="300px" align="right">
+</a>
 
-OpenThread implements all Thread networking layers including IPv6,
-6LoWPAN, IEEE 802.15.4 with MAC security, Mesh Link Establishment, and
-Mesh Routing.
+Want to try OpenThread? The quickest way to get started is to run through our [Simulation Codelab](https://codelabs.developers.google.com/codelabs/openthread-simulation/index.html), which covers all the basics, without the need for test hardware. Using VirtualBox and Vagrant on a Mac or Linux machine, you will learn:
 
+* How to set up the OpenThread build toolchain
+* How to simulate a Thread network
+* How to authenticate Thread nodes with Commissioning
+* How to use `wpantund` to manage a simulated Thread network featuring an NCP
 
-# Who is supporting OpenThread #
+### Next Steps
 
-![OpenThread Contributor Logos](doc/images/openthread_contrib.png)
+The Codelab shows you how easy it is use to OpenThread to simulate a Thread network.  Once complete:
 
-Nest, along with ARM, Atmel, a subsidiary of Microchip Technology,
-Dialog Semiconductor, Microsoft Corporation, Nordic Semiconductor,
-Qualcomm Technologies, Inc. (a subsidiary of Qualcomm Incorporated),
-and Texas Instruments Incorporated are contributing to the ongoing
-development of OpenThread.
+1. Learn more about the [OpenThread architecture and features](#openthread-features)
+1. Get familiar with [platforms and devices that support OpenThread](#who-supports-openthread)
+1. See what [testing tools](#what-tools-are-available-for-testing) are available
+1. Learn [where to get help](#need-help) and [how to contribute](#want-to-contribute) to the ongoing development of OpenThread
 
+# OpenThread Features
 
-# Getting started #
+OpenThread implements all features defined in the [Thread 1.1.1 specification](http://threadgroup.org/technology/ourtechnology#specifications), including all Thread networking layers (IPv6, 6LoWPAN, IEEE 802.15.4 with MAC security, Mesh Link Establishment, Mesh Routing) and device roles.
 
-The easiest way to get started is to run the CLI example in
-`/examples/apps/cli`. See the
-[CLI example README](examples/apps/cli/README.md)
-for more details.
+Other OpenThread-specific features and enhancements include:
 
+* Network authentication with Commissioning
+* IPv6 configuration and raw data interface
+* UDP sockets
+* CoAP client and server
+* DHCPv6 client and server
+* DNSv6 client
+* Command Line Interface (CLI)
+* Network Co-Processor (NCP) support - off-loading Thread functions to a SoC
+* Spinel - general purpose NCP protocol
+* `wpantund` - user-space NCP network interface driver/daemon
+* Sniffer support via NCP Spinel nodes
```

Comment:
Can we group the above items under: "Network Co-Processor Support"?  Maybe include a line somewhere like "OpenThread supports both single-on-chip (SoC) and network co-processor (NCP) designs."